README.md

```diff
@@ -31,29 +31,29 @@
 &nbsp;
 
 [Towhee](https://towhee.io) makes it easy to build neural data processing pipelines for AI applications.
-We provide several hundred models, algorithms, and transformations as standard pipeline building blocks.
-You can prototype your pipeline with our Pythonic API, and use Towhee to
+We provide several hundred models, algorithms, and transformations that can be used as standard pipeline building blocks.
+You can use Towhee's Pythonic API to build a prototype of your pipeline and
 automatically optimize it for production-ready environments.
 
-:art:&emsp;**Various Modalities:** We support data processing on different modalities, such as images, videos, text, audio, molecular structures, etc.
+:art:&emsp;**Various Modalities:** Towhee supports data processing on a variety of modalities, including images, videos, text, audio, molecular structures, etc.
 
-:mortar_board:&emsp;**SOTA Models:** We provide SOTA models across 5 fields (CV, NLP, Multimodal, Audio, Medical), 15 tasks, 140+ model architectures, 700+ pretrained models. These include BERT, CLIP, ViT, SwinTransformer, MAE, data2vec, etc.
+:mortar_board:&emsp;**SOTA Models:** Towhee provides SOTA models across 5 fields (CV, NLP, Multimodal, Audio, Medical), 15 tasks, and 140+ model architectures, including BERT, CLIP, ViT, SwinTransformer, MAE, and data2vec. All pretrained and 700+ more others to choose from.
```

Comment:
Towhee provides SOTA models across 5 fields (CV, NLP, Multimodal, Audio, Medical), 15 tasks, and 140+ model architectures. These include  BERT, CLIP, ViT, SwinTransformer, MAE, and data2vec, all pretrained and ready to use.

Reply:
Thank you for all the comments. It was especially hard for me to figure out the best way to rephrase the "pretrained" bit. Changes have been made in a new push.